package.json

```diff
@@ -9,9 +9,9 @@
     "benchmark": "npm run build && node -r source-map-support/register benchmarks/bench.js",
     "benchmark:debug": "npm run build && node -r source-map-support/register --inspect benchmarks/bench.js",
     "benchmark:prod": "npm run build:prod && node -r source-map-support/register benchmarks/bench.js",
-    "test": "npm run build && nyc mocha -r source-map-support/register --exit --timeout 20000 --parallel 'tests/**/*.test.js'",
-    "test:debug": "npm run build && mocha -r source-map-support/register --inspect --exit --timeout 20000 'tests/**/*.test.js'",
-    "test:prod": "npm run build:prod && nyc mocha -r source-map-support/register --exit --timeout 20000 'tests/**/*.test.js'",
+    "test": "npm run build && nyc mocha --parallel 'tests/**/*.test.js'",
```

Comment:
we cannot add the parallel also to the file ?

Reply:
That will make the debug difficult and probably not supported by most debugger client.

Reply:
Got it